bower.json

```diff
@@ -1,30 +1,53 @@
 {
   "name": "lazy-imports",
+  "version": "2.0.0",
```

Comment:
My thinking on this is that version numbers are doing double duty. They communicate to tools, and they communicate to people. Tools don't care if you jump from 0.0.z to 1.0.0 or 2.0.0, it's all the same to any semver constraints anyone was going to use. For the humans though, it'll be someone more useful I think for the hybrid version to have the same major version as our other hybrid elements, so my vote is for 2.0.0 as you have here.